presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -6408,4 +6410,40 @@ public void testPrefilterGroupByLimitNotFiredForLowCardinalityKeys()
         plan = computeActual(prefilter, "explain(type distributed) select count(custkey), orderkey from orders group by orderkey limit 100000");
         assertTrue(((String) plan.getOnlyValue()).toUpperCase().indexOf("MAP_AGG") == -1);
     }
+
+    @Test
+    public void testConversionOfCrossJoinWithLeftOuterJoinToInnerJoin()
+    {
+        Session useDefaultLiteralCoalesce = getSession();
```

Comment:
you can use a data provider to parametrize the test instead. Then just do regular list of assertQueries.  see example here: https://github.com/prestodb/presto/blob/master/presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java#L827-L850. What do you think?